lib/in/best-guess-homepage.js

```diff
@@ -1,17 +1,18 @@
 'use strict';
 
-const gitUrl = require('giturl');
+import gitUrl from 'giturl';
 
 function bestGuessHomepage(data) {
     if (!data) {
         return false;
     }
 
     const packageDataForLatest = data.versions[data['dist-tags'].latest];
-
+    /* eslint-disable no-mixed-operators */
     return packageDataForLatest.homepage ||
         packageDataForLatest.bugs && packageDataForLatest.bugs.url && gitUrl.parse(packageDataForLatest.bugs.url.trim()) ||
         packageDataForLatest.repository && packageDataForLatest.repository.url && gitUrl.parse(packageDataForLatest.repository.url.trim());
+    /* eslint-enable no-mixed-operators */
```

Comment:
I think that we can wrap with parenthesis instead of disabling the rule:

```suggestion
    return packageDataForLatest.homepage ||
        (packageDataForLatest.bugs && packageDataForLatest.bugs.url && gitUrl.parse(packageDataForLatest.bugs.url.trim())) ||
        (packageDataForLatest.repository && packageDataForLatest.repository.url && gitUrl.parse(packageDataForLatest.repository.url.trim()));
```